### PR TITLE
[docs][yba] DOC-92 multi-cloud deployment

### PR DESCRIPTION
**Pull request by ddhodge**
Clarify IP addresses of nodes in multi-cloud deployment, update to new UI

@netlify /preview/yugabyte-platform/create-deployments/create-universe-multi-cloud/

### CODE REVIEW

**Review by shinji62 — APPROVED** [review]

**Comment by shinji62** [comment]
Small nitpick not sure about those  being set up as Gflags
```
use_cassandra_authentication=true - Deployments on public clouds require security.
ysql_enable_auth=true - Deployments on public clouds require security.
```

those should be set with we select enable auth in the YBA ui no ?

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-cloud.md [hunk 1]
```diff
@@ -64,47 +64,34 @@ You can configure the on-premises cloud provider for YugabyteDB using YugabyteDB
...
-
-  ![caption](/images/ee/multi-cloud-provider-info.png)
-
-### Define an instance type
-
-Select **Instance Types** and enter a machine description which matches the nodes you will be using. The machine type can be any logical name, given the machine types will be different between all three regions. The following example uses `8core`:
-
-![Multi-cloud instance description](/images/ee/multi-cloud-instances.png)
+* **DB Nodes have public internet access?** should be enabled if your nodes have Internet connectivity (that is, you are not performing an airgap install).
+* Click **Add Region** atond define your regions and availability zones.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-cloud.md:82]:
```suggestion
* Click **Add Region** to define your regions and availability zones.
```

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-cloud.md [hunk 2]
```diff
@@ -64,47 +64,34 @@ You can configure the on-premises cloud provider for YugabyteDB using YugabyteDB
...
 
-### Define regions
+Click **Create Provider Configuation** when done.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-cloud.md:84]:
```suggestion
Click **Create Provider Configuration** when done.
```

**Review by aishwarya24 — APPROVED** [review]
minor comments

**Pull request merged**